src/web/resources/report-page.js

```diff
@@ -606,24 +600,11 @@ function buildControls(jsonData, diffCount) {
             `, compared with baseline `, Element("code", {}, otherJsonData.branch),
         ]])
 
-    const input = Element("input", {
-        id: "compare-input", value: compareAgainstURL,
-        placeholder: "URL to other json file"
-    }, [])
-
-    var showToleranceBool = false
-    if (radioState == "time" ||
-        radioState == "targetAccuracy" ||
-        radioState == "resultAccuracy" ||
-        radioState == "startAccuracy") {
-        showToleranceBool = true
-    }
+    const toleranceInputField = showTolerance(jsonData)
```

Comment:
What value does show default to if nothing is passed in for it?

Reply:
This change just moved a bunch of code from `buildFormControls` into `showTolerance`